src/main/java/com/rakovets/course/java/core/practice/operators/Task03.java

```diff
@@ -35,8 +34,7 @@ public static void main(String[] args) {
      * @param depositAnnualPercentage ежегодный простой процент
      */
     static float getTotalDepositAmount(int depositAmount, int depositYears, int depositAnnualPercentage) {
-        //TODO
-        // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return 0.0f;
+
```

Comment:
"Violation: Useless blank lines
Solution: Remove blank lines"

Reply:
this fixed